CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Contributing to fiano
+
+Contributions are very welcome. See [issues](https://github.com/linuxboot/fiano/issues), and join us in [Slack](CONTRIBUTING.md#communication) to talk about your cool ideas for the project.
+
+## Developer Sign-Off
+
+For purposes of tracking code-origination, we follow a simple sign-off
+process.  If you can attest to the [Developer Certificate of
+Origin](https://developercertificate.org/) then you append in each git
+commit text a line such as:
+```
+Signed-off-by: Your Name <username@youremail.com>
+```
+## Code of Conduct
+
+Conduct your collaboration around the u-root [Code of
+Conduct](https://github.com/u-root/u-root/wiki/Code-of-Conduct).
+
+## Communication
+
+- [Slack](https://u-root.slack.com), sign up
+[here](http://slack.u-root.com/)
+- [Join the mailing list](https://groups.google.com/forum/#!forum/u-root)
+
+## Bugs
+
+- Please submit issues to https://github.com/linuxboot/fiano/issues
+
+## Coding Style
+
+This project aims to follow the standard formatting recommendations
+and language idioms set out in the [Effective Go](https://golang.org/doc/effective_go.html)
+guide, for example [formatting](https://golang.org/doc/effective_go.html#formatting)
+and [names](https://golang.org/doc/effective_go.html#names).
+
+We have a few rules not covered by these tools:
+
+- Standard imports are separated from other imports. Example:
+    ```
+    import (
+      "regexp"
+      "time"
+
+      dhcp "github.com/krolaw/dhcp4"
```

Comment:
this is going to change soon! :P

Reply:
it's not important here, it's just an example.

Reply:
I know, I was just joking

Reply:
If I'm not mistaken this is what goimports does as a formatter instead of gofmt right ?